src/renderer/app/index.html

```diff
@@ -11,7 +11,8 @@
     <meta name="msapplication-TileColor" content="#da532c">
     <meta name="theme-color" content="#ffffff">
 
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <!--    This tag commented on purpose - that's the way to "support" mobile devices-->
+    <!--    <meta name="viewport" content="width=device-width, initial-scale=1" />-->
```

Comment:
Maybe delete it then? If we aren't gonna use it

Reply:
It's very very very default tag for every web app, we should uncomment it when more responsive design will be delivered